plugins/modules/activation_key.py

```diff
@@ -82,7 +82,7 @@
   content_overrides:
     description:
       - List of content overrides that include label and override state
-      - Label refers to repository C(content_label), e.g. rhel-7-server-rpms
+      - Label refers to repository in the format: <organizationLabel>_<productLabel>_<repositoryLabel>, e.g. ExampleOrg_ExampleProduct_ExampleRepository
```

Comment:
That's correct for custom products, but not for Red Hat ones, where it is `rhel-7-server-rpms` etc. That's why `content_label` is referred here, as that's how Katello calls it and it can differ. Should we do:
```suggestion
      - Label refers to repository C(content_label).
      - For Red Hat products for example C(rhel-7-server-rpms).
      - For custom products it's in the format: C(<organization_label>_<product_label>_<repositoryLabel>), e.g. C(ExampleOrg_ExampleProduct_ExampleRepository).
```

Reply:
Thanks for explaining. I my case it was custom product. 
Your change makes it more clear for both situations.
Thanks.